scripts/bump-version.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Script to bump version in all required files
+# Usage: ./bump-version.sh <new_version>
+# Example: ./bump-version.sh 2.5.0
+
+if [ $# -eq 0 ]; then
+    echo "Usage: $0 <new_version>"
+    echo "Example: $0 2.5.0"
+    exit 1
+fi
+
+NEW_VERSION=$1
+
+# Validate version format
+if ! [[ $NEW_VERSION =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    echo "Error: Version must be in format x.y.z (e.g., 2.5.0)"
+    exit 1
+fi
+
+echo "Updating version to $NEW_VERSION..."
+
+# Get current version
+CURRENT_VERSION=$(grep -Po '"version": "\K[0-9]+\.[0-9]+\.[0-9]+' package.json)
+echo "Current version: $CURRENT_VERSION"
+
+# Update package.json
+echo "Updating package.json..."
+sed -i.bak "s/\"version\": \"$CURRENT_VERSION\"/\"version\": \"$NEW_VERSION\"/" package.json
```

Comment:
Wondering if you can run https://formulae.brew.sh/formula/shellcheck for the formatting